geode-serialization/src/test/java/org/apache/geode/internal/serialization/filter/ReflectiveFacadeGlobalSerialFilterFactoryTest.java

```diff
@@ -57,8 +66,9 @@ public void delegatesToObjectInputFilterApiToCreateObjectInputFilter()
   public void delegatesToObjectInputFilterApiToSetSerialFilter()
```

Comment:
Similar comments to the previous test re: the name and testing an implementation detail.

Reply:
Fixed up ReflectiveFacadeGlobalSerialFilterFactoryTest

Reply:
Note: I removed tests from ReflectiveFacadeGlobalSerialFilterFactoryTest that were redundant with tests in ReflectiveFacadeGlobalSerialFilterTest and *ApiTest.